### ISSUE TITLE
VSP = Filter?

### ISSUE THREAD

**Issue by soupman98** [maartenvantjonger/omnilogic-smartapp#12]
When I re-added my BOW VSP (Variable Speed Pump) it same in a <bow name> Filter" (e,g, Swimin Hol Filter). Seems like it should be pump?

**Comment by soupman98** [comment]
Looking at the code and thinking about the install - I guess i kinda makes sense.. the VSP is on the filter, where as the extra pumps are on features typically. and running off filtered water....

**Comment by maartenvantjonger** [comment]
Yeah, technically it's a pump like all the others but the VSP driver is filter specific in the in the way it updates its status and it offers spillover (From spa to main pool for example).
I could call it `<Pool name> Filter Pump` by default instead of `<Pool name> Filter`. What do you think?

Regardless, you can rename the device to whatever you like in the Smartthings App, as long as the device ID remains the same.
